### PR TITLE
chore: rerun light_ecModel tutorial

### PR DESCRIPTION
**Pull request by edkerk**
### Main improvements in this PR:
- fix:
  - `getSubsetEcmodel` #327
  - `plotlightVSfull` #332 
- chore:
  - rerun the complete tutorial including gathering data from databases and running DLKcat. Rerunning might again result in different results, as e.g. databases have changed.
- doc:
  - reformat warnings to be clearly marked with ´WARNING:` and printed in orange in GUI.

**I hereby confirm that I have:**

- [x] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

File: tutorials/full_ecModel/code/plotlightVSfull.m [hunk 1]
```diff
@@ -58,12 +58,15 @@
 fprintf('Growth rate that is reached: %.4f vs %.4f\n', abs(solFull.f) , abs(solLight.f))
 
 %
+solF(abs(solF)<1e-10) = 0;
```
[the remark this answers is not in the document]
Comment (edkerk) [reply tutorials/full_ecModel/code/plotlightVSfull.m]: Fair point, now made consistent.

**Review by ae-tafur — APPROVED** [review]
For me looks ok but I understand than @mihai-sysbio is also reviewing.

**Comment by mihai-sysbio** [comment]
Thanks for the review and approval @ae-tafur 

@edkerk feel free to merge if you can't wait anymore :) anything I picked up, I'll create a separate issue for.

**Comment by edkerk** [comment]
Maybe we can make a release, to check that the `doc` becomes available online and Zenodo picks up the release?

**Pull request merged**